DataFormats/L1THGCal/interface/HGCFETriggerDigi.h

```diff
@@ -51,7 +51,7 @@ namespace l1t {
     template<typename IDTYPE>
     IDTYPE getDetId() const { return IDTYPE(detid_); }
     template<typename IDTYPE>
-    void   setDetId(const IDTYPE& id) { detid_ = id.uint32_t(); }
+    void   setDetId(const IDTYPE& id) { detid_ = id(); }
```

Comment:
You can call it like a function too, it's just idiomatic that you can also do (uint32_t) to convert.